File: Daily/February/LeetCode1748.cc
#include<iostream>
#include<vector>
#include<algorithm>
#include<unordered_map>
using namespace std;

class Solution {
public:
    int sumOfUnique(vector<int>& nums) {
        unordered_map<int,int> cnt;
        int res = 0;
        for(const int &num: nums){
            ++cnt[num];
        }
        for(auto i: cnt){
            if(i.second == 1){
                res += i.first;
            }
        }
        return res;
    }
};

int main(){
    vector<int>nums = {1,1,1,1,1};
    unordered_map<int,int> cnt;
    int res = 0;
    for(const int &num: nums){
        ++cnt[num];
    }
    
    for(auto x : cnt){
        cout << "Key:[" << x.first << "] Value:[" << x.second << "]\n";
    }
    cout << " "<<endl;
}

File: Daily/February/LeetCode1748.go
package february

func sumOfUnique(nums []int) int {

	state := map[int]int{}
	ans := 0
	for _, num := range nums {
		if state[num] == 0 {
			ans += num
			state[num] = 1
		} else if state[num] == 1 {
			ans -= num
			state[num] = 2
		}
	}
	return ans
}
